packages/styles/src/components/_index.scss

```diff
@@ -45,4 +45,4 @@
 @use 'datepicker';
 @use 'dropdown';
 @use 'floating-label';
-@use 'form-feedback';
+@use 'form-validation';
```

Comment:
Is this part of this PR or should we move it to an own PR?

Reply:
This PR is to merge back changes from this branch: https://github.com/swisspost/design-system/compare/main...re-applied-icon-changes. It includes the renaming of "form-feedback" to "form-validation".